backend/src/junior/junior.service.ts

```diff
@@ -246,9 +246,19 @@ export class JuniorService {
         return next;
     }
 
-    async createNewSeason(): Promise<string> {
+    async createNewSeason(expireDate: SeasonExpiredDto): Promise<string> {
         const result: UpdateResult = await this.juniorRepo.createQueryBuilder().update().set({ status: "expired" }).execute()
-        return `New season created. ${result.affected} juniors expired`;
+
+        const listJunior = await this.juniorRepo.find();
+
+        let smsFailure: string[] = []
+
+        await Promise.all(listJunior.map(async (junior: Junior) => {
+            const messageSent = await this.smsService.sendNewSeasonSMS({ name: `${junior.firstName} ${junior.lastName}`, phoneNumber: junior.parentsPhoneNumber }, expireDate.expireDate);
+            if (!messageSent) { smsFailure.push(junior.parentsPhoneNumber) }
```

Comment:
The phone numbers with failed messages should also be logged for easier debugging. For example,
```
console.warn(`Failed to send SMS to ${junior.parentsPhoneNumber}`)
```